napari/layers/base/base.py

```diff
@@ -395,7 +395,7 @@ def _mode_setter_helper(self, mode, Modeclass):
             callback_list.append(mode_dict[mode])
         self.cursor = self._cursor_modes[mode]
 
-        self.interactive = mode == Modeclass.PAN_ZOOM
+        self.interactive = mode in {Modeclass.PAN_ZOOM, Modeclass.FILL_CONTOUR}
```

Comment:
This is creating most of the test failures as not all modeclass have a  FILL_CONTOUR.

```suggestion
        self.interactive = mode in {Modeclass.PAN_ZOOM, getattr(Modeclass, "FILL_CONTOUR", object())}
```

Just a band aid to get most of the test passing and see if it has any other side effects. We might want to decide whether  the `Modeclass` need to have a `_interactive_modes` set, so that we can do something like:

```
self.interactive = mode in Modeclass._interactive_modes
```

Reply:
Going to commit suggestion just to see if test are passing. Feel free to revert modify.